Chummer/Controls/SustainedSpellControl.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+using System.Xml;
+using System.Xml.XPath;
+using Chummer.Backend.Equipment;
+using Chummer.Backend.Uniques;
+
+namespace Chummer
+
+{
+    public partial class SustainedSpellControl : UserControl
+    {
+        private readonly ISustainable _objSustainedSpell;
+        private bool _blnLoading = true;
+
+        //Events
+        public event EventHandler SpellDetailChanged;
+        public event EventHandler UnsustainSpell;
+
+
+        public SustainedSpellControl(ISustainable objSustainedSpell)
+        {
+            _objSustainedSpell = objSustainedSpell;
+            InitializeComponent();
+            this.UpdateLightDarkMode();
+            this.TranslateWinForm();
+        }
+
+        private void SustainedSpellControl_Load(object sender, EventArgs e)
+        {
+            lblSustainedSpell.Text = _objSustainedSpell.DisplayName(GlobalOptions.Language);
+            nudForce.DoDatabinding("Value", _objSustainedSpell, nameof(_objSustainedSpell.Force));
+            nudNetHits.DoDatabinding("Value", _objSustainedSpell, nameof(_objSustainedSpell.NetHits));
+
+            //Prevents the user from setting  SelfSustained of an SustainedCritterPower to true, because sustained critterpowers don't cause any dice loss
+            if (_objSustainedSpell is SustainedCritterPower)
+            {
+                chkSelfSustained.Visible = false;
+                lblSelfSustained.Visible = false;
+            }
```

Comment:
Would this not work better in the constructor where `InitializeComponent()` is already? `_objSustainedSpell` is not supposed to change after being set in the constructor anyway, so it's not like there's a risk of breaking data bindings by hiding these from the very beginning.